core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -187,6 +188,21 @@ default void waitTopicDeletion(String topic) throws InterruptedException {
         waitForTopic(topic, 0);
     }
 
+    default long waitForMeatdataSync() throws InterruptedException {
```

Comment:
It is totally unused. Why we need this helper?

Reply:
Yes, it is not used now.
This comes from `TestUtils.ensureConsistentKRaftMetadata` and I think it can help new infra user easy to sync metadata.
WDYT?

Reply:
Please add test at least